src/lib/preferences.ts

```diff
@@ -66,21 +75,47 @@ export interface Preferences {
   'markdown.previewStyle': string
   'markdown.codeBlockTheme': string
   'markdown.includeFrontMatter': boolean
+
+  // Keymap
+  'general.keymap': Map<string, KeymapItem> | null
```

Comment:
I'd like to fill an empty map rather than allowing `null` type. Otherwise, we need to do null-checking everywhere using this attribute.

Reply:
Refactored